docs/README.md

```diff
@@ -50,7 +50,7 @@ Once you are comfortable with the GDK and want to start a separate project, look
 * [Sending and receiving events](content/gameobject/sending-receiving-commands.md)
 * [Sending and receiving commands](content/gameobject/sending-receiving-events.md)
 * [World commands](content/gameobject/world-commands.md)
-* [Logging and accessing the worker](content/gameobject/logging-and-accessing-the-worker.md)
+* [SpatialOSComponent utility API](content/gameobject/spatialoscomponent-utility-api.md)
```

Comment:
I'd say this is a worse title as I, as a user, have no idea what the SpatialOSComponent utility API could do for me. 
Logging and accessing the worker is helpful as I know why I might want to read the linked doc.

Reply:
I see the point about logging, but you should find a name more in line with "Sending and reciving events" centric to the task the user will have in mind.

Reply:
The main problem is with the design of the component itself in the first place imo. It's just a place where a bunch of unrelated things ended up, the previous title was not really describing the contents. For now I think it's OK like this and when we hopefully break it up we can make the docs make more sense too.

Reply:
Agreed. Once we can properly inject your entityid or the logger, this component will have a clearer role or will be removed altogether. The current state is more of a stop gap.

Reply:
Renamed to `Retrieving your EntityId and accessing the worker` for now.